### DISCUSSION TITLE
add draggable="false" in Link and NextImage props

### DISCUSSION THREAD

**Discussion by EsrafilElahi** [vercel/next.js#73632] [Ideas]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
in next all apps, all versions
```

### Which example does this report relate to?

with-tailwindcss , all examples

### What browser are you using? (if relevant)

firefox

### How are you deploying your application? (if relevant)

the compony own server

### Describe the Bug

to preventing draggable in some browsers such as firefox :
the Link and NextImage tags must be include draggable props
this helps for example Sliders components which don't throw it

### Expected Behavior

accept the passing draggable="false" in Link and Image tag props
doesnt type error

### To Reproduce

to preventing draggable in some browsers such as firefox :
the Link and NextImage tags must be include draggable props
this helps for example Sliders components which don't throw it

**Comment by imLymei** [comment]
Both components have the `draggable` prop, but it accepts a Booleanish value.
![image](https://github.com/user-attachments/assets/b37f3635-3b55-431e-97a4-35569e338942)

**Reply by EsrafilElahi** [reply]
yes , those have draggable props
i am using nextjs 12  , in this version not exists
thank you by the way
